packages/tools/src/utilities/lengthUnits.ts

```diff
@@ -0,0 +1,19 @@
+const lengthUnits = (handles, image): string => {
```

Comment:
since these two functions involve looking at the calibration, I prefer other namings suh as `calibratedLenghUnit` (btw why units?), or `getCalibratedLengthUnit` same for area